models/account.js

```diff
@@ -16,6 +16,7 @@ const Account = new Schema({
 		staffDisableVisibleElo: Boolean,
 		staffDisableStaffColor: Boolean,
 		staffIncognito: Boolean,
+		staffEditorCustomColour: String,

```

Comment:
This really ought to keep a consistent naming scheme with `staffDisableStaffColor` - i.e. `staffEditorCustomColor`